ApplicationPattern.yaml

```diff
@@ -1632,6 +1731,7 @@ paths:
                 oam-approval-application: 'AdministratorAdministration'
                 oam-approval-application-release-number: '2.0.0'
                 oam-approval-operation: '/v1/approve-oam-request'
+                oam-approval-protocol: 'HTTP'
```

Comment:
line 1786: callback uri definition for OamRequestCausesInquiryForAuthentication has protocol https which needs to be updated to reference oam-approval-protocol {$request.body#oam-approval-protocol}://...

Reply:
Changed as proposed. Many thanks for the concrete proposal.